proto/v2/v2.proto

```diff
@@ -19,11 +19,13 @@ package datacommons.v2;
 
 import "entity.proto";
 
-message Arc {
-  // The property of one single arc. This is only a single property, rather than
-  // chained properties
-  string property = 1;
-  repeated datacommons.EntityInfo nodes = 2;
+message Nodes {
+  repeated datacommons.EntityInfo nodes = 1;
+}
+
+message Arcs {
+  map<string, Nodes> arcs = 2;
+  map<string, Arcs> neighbor = 3;
```

Comment:
Where is the poor 1? :)

Reply:
The naming is a bit confusing, especially the first one, the map val type is 'Nodes', but the variable name is 'arcs'. How about:

  map<string, Nodes> neighbor_nodes = 1;
  map<string, Arcs> neighbor_arcs = 2;

Or, just:

  map<string, Nodes> nodes = 1;
  map<string, Arcs> arcs = 2;

Reply:
I initially tried this too, but it does not look right. I just used this to update the golden and pushed and you can see the actual response. By comparing with the previous version, you can tell it was more readable.

Basically, "neighbor" is to recursive to next level of "arcs" and "arcs" is linked to leaf "nodes".